### PR TITLE
LPS-179513 Apply new design

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by sandrodw3** [comment]
ci:test:sf

**Comment by sandrodw3** [comment]
ci:test:sf

File: modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/logo_selector/LogoSelector.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+						id={`${portletNamespace}logoName`}
+						readOnly
+						value={logoName || Liferay.Language.get('custom-x')}
+					/>
+
+					<ClayButtonWithIcon
+						aria-label={sub(
+							Liferay.Language.get('change-x'),
+							label
+						)}
```
Comment (p2kmgcl) [inline modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/logo_selector/LogoSelector.js:125]: I think we can omit this if we are adding a title attribute, can't we?

File: modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/logo_selector/LogoSelector.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+						className="flex-shrink-0 mr-2"
+						displayType="secondary"
+						onClick={onChangeLogo}
+						symbol="change"
+						title={sub(Liferay.Language.get('change-x'), label)}
+					/>
+
+					<ClayButtonWithIcon
+						aria-label={sub(Liferay.Language.get('clear-x'), label)}
```
Comment (p2kmgcl) [inline modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/logo_selector/LogoSelector.js:134]: Same as before, I think we can omit this

**Review by p2kmgcl — COMMENTED** [review]
Very nice @sandrodw3. I added a couple of doubts I have about a global function and an accessibility concern, but feel free to ignore those comments if I am wrong :smile:

File: modules/apps/image-uploader/image-uploader-web/src/main/resources/META-INF/resources/upload_image.jsp [hunk 3]
```diff
@@ -41,10 +41,22 @@ String tempImageFileName = ParamUtil.getString(request, "tempImageFileName");
...
+						'<%= fileEntry.getFileEntryId() %>'
+					);
+				}
+				else {
+					Liferay.Util.getOpener().Liferay.fire('changeLogo', {
+						previewURL: '<%= previewURL %>',
+						fileEntryId: '<%= fileEntry.getFileEntryId() %>',
+						tempImageFileName: '<%= HtmlUtil.escape(tempImageFileName) %>',
+					});
+				}
```
[the remark this answers is not in the document]
Comment (sandrodw3) [reply modules/apps/image-uploader/image-uploader-web/src/main/resources/META-INF/resources/upload_image.jsp:59]: I left the current behavior so the current usages of the old taglib continue working, as they are binding the `changeLogo` functions in aui scripts and not listening to any event 😞 when we migrate all usages to the new taglib I think we will be able to remove this 😄

File: modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/logo_selector/LogoSelector.js (hunk 1 above)
Comment (sandrodw3) [reply modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/logo_selector/LogoSelector.js:125]: You are right, I think we can remove it if it's not giving any extra info, updating it :rocket:

File: modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/logo_selector/LogoSelector.js (hunk 2 above)
Comment (sandrodw3) [reply modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/logo_selector/LogoSelector.js:134]: Same :rocket:

File: modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/logo_selector/LogoSelector.js (hunk 1 above)
Comment (sandrodw3) [reply modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/logo_selector/LogoSelector.js:125]: Um, I think we have to put both `title` and `aria-label` if the button does not have any text 🤔 I'll check it with Vero

File: modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/logo_selector/LogoSelector.js (hunk 2 above)
Comment (sandrodw3) [reply modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/logo_selector/LogoSelector.js:134]: Same :thinking:

File: modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/logo_selector/LogoSelector.js (hunk 1 above)
Comment (sandrodw3) [reply modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/logo_selector/LogoSelector.js:125]: She told me we are putting both for those cases 😄 so I'm leaving them

File: modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/logo_selector/LogoSelector.js (hunk 2 above)
Comment (sandrodw3) [reply modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/logo_selector/LogoSelector.js:134]: Leaving both as Vero suggested :rocket:

**Comment by ealonso** [comment]
see: https://github.com/liferay-echo/liferay-portal/pull/11826

**Pull request closed without merging**